src/data_warehouse/core/storage/__init__.py

```
@@ -0,0 +1,46 @@
+"""Storage functionality for the data warehouse."""
+
+from data_warehouse.core.exceptions import DatabaseError, StorageError
+from data_warehouse.utils.logger import logger
+
+# Import storage components
+from .duckdb import (
+    DuckDBClient,
```

Ruff (0.8.2)

`.duckdb.DuckDBClient` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)